base/uk.ac.stfc.isis.ibex.ui.synoptic/src/uk/ac/stfc/isis/ibex/ui/synoptic/SynopticSelectionDialog.java

```diff
@@ -74,10 +75,11 @@ protected void createSelection(Composite container) {
 		Label lblSelect = new Label(container, SWT.NONE);
         lblSelect.setText("Select synoptic:");
 		
-        items = createTable(container, SWT.BORDER | SWT.V_SCROLL);
+        List<String> columnNames = Arrays.asList("Name");
+        items = createTable(container, SWT.BORDER | SWT.V_SCROLL, columnNames);
 
-		String[] names = SynopticInfo.names(available).toArray(new String[0]);
-		Arrays.sort(names, String.CASE_INSENSITIVE_ORDER);
+		List<String> names = (List<String>) SynopticInfo.names(available);
+		names.sort(String.CASE_INSENSITIVE_ORDER);
```

Comment:
As above, API issue with unordered vs ordered colelctions except I think this one is solvable with something like

```suggestion
		SynopticInfo.names(available).stream()
		    .sorted(String.CASE_INSENSITIVE_ORDER)
		    .collect(Collectors.toList());
```